src/lib/decodeData.test.ts

```diff
@@ -42,13 +42,27 @@ describe('decodeData', () => {
       valueType: 'bytes',
       valueContent: 'JSONURL',
     },
+    {
+      name: 'MyKeyName2:<bytes32>:<bool>',
+      key: '0x',
+      keyType: 'Singleton',
```

Comment:
I am not sure why, these two schemas are not `Singleton` keys. But maybe it does not matter in this test?

<img width="441" alt="image" src="https://github.com/ERC725Alliance/erc725.js/assets/31145285/96a26dda-7078-4426-8532-97e692a9acdf">


```suggestion
      keyType: 'MappingWithGrouping',
```

Reply:
I swapped these